pkg/storage/snapshot/restore.go

```diff
@@ -562,6 +564,11 @@ func (t *vmRestoreTarget) Reconcile() (bool, error) {
 }
 
 func (t *vmRestoreTarget) reconcileBackendVolume() (bool, error) {
+	if !t.Exists() {
```

Comment:
do we even want to allow clone if backend storage exists? wouldnt it create a vm which is not usable?

Reply:
That's a good point and something we need to discuss further. There was some conversation about this last Friday with @awels and @mhenriks, and we decided to check what other VM solutions do in similar cases, but the general idea semed to be that cloning doesn't seem like a valid use case outside of testing and we probably shouldn't enable it. We still need to decide whether it's better to block the operation or simply ignore the TPM. I'll look into this further once we merge this PR/online snapshot, but I'm not sure it's worth investing too much time into.